### PR TITLE
Updated SNAPTEL.md - added mention about separated snap-cli repo

### PR DESCRIPTION
**Pull request by IzabellaRaulin**
Summary of changes:
-  updated SNAPTEL.md - added mention about separated snap-cli repo

@intelsdi-x/snap-maintainers

### CODE REVIEW

File: docs/SNAPTEL.md [hunk 1]
```diff
@@ -18,16 +18,48 @@ limitations under the License.
...
+    The `snaptel` which bases on REST API V1 is available in the main Snap repo, as a part of
+[Snap releases](https://github.com/intelsdi-x/snap/releases) and `snap-telemetry` package.
+
+
+ - **Snap CLI on top of [REST API V2](REST_API_V2.md)**
+
+    The _alpha version_ of `snaptel` which bases on REST API V2 is available in separated repo
+[snap-cli](https://github.com/intelsdi-x/snap-cli).
+
+    The separated Snap API client and CLI support [Swagger Open API Specification](https://swagger.io/specification). To get more help on the Snap API client and Snap CLI check out:
```
Comment (andrzej-k) [inline docs/SNAPTEL.md]: This section talks about "separated Snap API client" which is related yet different from Snap CLI - maybe consider a small sub-section explaining new Snap API client, or just putting link where it is described? Maybe a bit of introduction saying that new CLI is based on new Snap API client which in turn is talking to API v2 would be good?

File: docs/SNAPTEL.md [hunk 2]
```diff
@@ -206,5 +238,9 @@ $ snaptel plugin unload processor passthru <version>
 $ snaptel plugin unload publisher publisher <version>
 ```
 
-## More information
+### More information
 * [SECURE_PLUGIN_COMMUNICATION](SECURE_PLUGIN_COMMUNICATION.md)
+
+## Snap CLI (REST API V2)
+The documentation for the Snap API CLI which bases on REST API V2 is located in [snap-cli](https://github.com/intelsdi-x/snap-cli).
```
Comment (andrzej-k) [inline docs/SNAPTEL.md]: "Snap API CLI" -> "Snap CLI"

Comment (IzabellaRaulin) [reply docs/SNAPTEL.md]: addressed

**Comment by kjlyon** [comment]
@IzabellaRaulin , in this PR, can you also address this issue: 

In the example section of SNAPTEL.md, when unloading the plugins, there are two typos. 
- `mock` needs to be changed to `mock1`
- and the second `publisher` needs to be changed to `mock-file`
image

**Comment by IzabellaRaulin** [comment]
@kjlyon, according:
> the second publisher needs to be changed to mock-file
Thanks for catching that. I will change it.

> mock needs to be changed to mock1
snap-plugin-collector-mock1 has a name `mock` and version `1` (the version has never changed because the plugin is used in tests together with snap-plugin-collector-mock2 to check version swapping), so the name="mock" in this case is as expected

**Comment by kjlyon** [comment]
LGTM

File: docs/SNAPTEL.md [hunk 3]
```diff
@@ -18,16 +18,45 @@ limitations under the License.
...
+
+### Snap CLI on top of REST API V1
+
+The `snaptel` which bases on [REST API V1](REST_API_V1.md) is available in the main Snap repo, as a part of
+[Snap releases](https://github.com/intelsdi-x/snap/releases) and `snap-telemetry` package.
+
+
+### Snap CLI on top of REST API V2
+
+The _alpha version_ of `snaptel` which relays on [Snap API Go client](https://github.com/intelsdi-x/snap-client-go) talking to [REST API V2](REST_API_V2.md) is available in separated repo [snap-cli](https://github.com/intelsdi-x/snap-cli).
```
Comment (rashmigottipati) [inline docs/SNAPTEL.md]: Minor correction: 
in separated repo --> in a separate repo

**Pull request merged**